src/__test__/upload.test.ts

```diff
@@ -0,0 +1,74 @@
+import o from 'ospec';
+import sinon from 'sinon';
+import { BackOff, uploadFile } from '../upload';
+
+o.spec('Upload', () => {
+  let sandbox: sinon.SinonSandbox;
+
+  o.before(function () {
+    sandbox = sinon.createSandbox();
```

Comment:
I think you can create the sandbox once on L6 dont need to make a newone in the before